coverage_utils.py

```diff
@@ -10,26 +11,54 @@
 from core.metadata_layer import ReplacementPolicy
 from core.mirror import MirrorUploader
 from core.model import (
+    get_one,
     DataSource,
+    ExternalIntegration,
+    ExternalIntegrationLink,
     Work,
 )
 
+class MetadataWranglerReplacementPolicy(ReplacementPolicy):
+    """A ReplacementPolicy that uses the only configured storage
+    integration as its cover mirror.
+    """
+    @classmethod
+    def from_db(cls, _db, **kwargs):
+        """Create a MetadataWranglerReplacementPolicy based on
+        database settings -- specifically, with the configured
+        MirrorUploader for cover images.
+
+        :param mirror: Pass in a mock MirrorUploader to use it
+        instead of creating a real one.
+        """
+        mirror = kwargs.pop('mirror', None)
+        if not mirror:
+            integration = get_one(
```

Comment:
Won't this cause an issue if there are more than one ExternalIntegrations with the storage goal?

Reply:
On the metadata wrangler, having more than one storage integration _is_ the problem -- we don't currently know what that would mean in real-world terms. If there's no storage integration, then I think the best thing is to log an error and proceed -- the storage integration isn't required. If there are two storage integrations, then I really don't know what should happen, and I think we should stop work until the situation is resolved.